File: OOP/geo/Point.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace geo
{
    public class Point
    {
        private double x { get; set; }
        private double y { get; set; }

        public Point(double x, double y)
        {
            this.x = x;
            this.y = y;
        }

        public double Distane(Point p)
        {
            return  Math.Sqrt(Math.Pow( x - p.x, 2 ) + Math.Pow( y - p.y, 2 ));
        }
    }
}

File: Java/2two/Point.java
public class Point{
    double x;
    double y;

    public Point(double x, double y){
        this.x = x;
        this.y = y;
    }

    public void move (double x, double y){
        this.x += x;
        this.y += y;
    }

    public void mirror(Point p){
        x = x - p.x - p.x;
        y = y - p.y - p.x;
    }

    public double Distane(Point p)
    {
        return  Math.sqrt(Math.pow( x - p.x, 2 ) + Math.pow( y - p.y, 2 ));
    }
}